compat-data-schema.md

```diff
@@ -0,0 +1,365 @@
+# The browser-compat-data JSON format
```

Comment:
Also the README should be updated to point to this document: at the moment it documents the old schema.